src/controllers/ownerMessageController.js

```diff
@@ -15,7 +18,10 @@ const ownerMessageController = function (OwnerMessage) {
   };
 
   const updateOwnerMessage = async function (req, res) {
-    if (req.body.requestor.role !== 'Owner') {
+    if (
+      req.body.requestor.role !== 'Owner' &&
+      !(await helper.hasPermission(req.body.requestor, 'editHeaderMessage'))
+    ) {
```

Comment:
```suggestion
    if (
      !(await helper.hasPermission(req.body.requestor, 'editHeaderMessage'))
    ) {
```
I think the permissions are supposed to be replacing role logic entirely. You add 'editHeaderMessage' in `createInitialPermissions.js`, so Owners should always have this permission. Not requiring them to be an owner allows for the header message to perhaps be delegated to a non-owner account.